functions/influxdb+grafana.bash

```diff
@@ -1,359 +1,297 @@
 #!/usr/bin/env bash
 
-## Function for installing and configure InfluxDB and Grafana while also integrate it to openHAB.
-## The function can be invoked either INTERACTIVE with userinterface UNATTENDED.
-##
-## When called UNATTENDED it will install both InfluxDB and Grafana on the local system.
+## Function to install and configure InfluxDB and Grafana, and integrate them with openHAB.
+## This function can only be invoked in INTERACTIVE with userinterface.
 ##
 ##    influxdb_grafana_setup()
 ##
-
 influxdb_grafana_setup() {
-  local FAILED
-  local textIntro textUnsupported textLowMem textFail textFailLowMem textSuccess
-  local influxDBAddress influxDBAdminUsername influxDBAdminPassword
-  local influxDBopenhabUsername influxDBopenhabPassword
-  local influxDBGrafanaUsername influxDBGrafanaPassword
+  if [[ -z $INTERACTIVE ]]; then
+    echo "$(timestamp) [openHABian] InfluxDB and Grafana setup must be run in interactive mode! Canceling InfluxDB and Grafana setup!"
```

Comment:
Correct for the time being but should not be hard to provide a version that also does unattended install taking its values off openhabian.conf, should it ....

Reply:
No, if that becomes a desired feature it should be fairly simple.

Reply:
as mentioned earlier we should provide interactive AND unattended for every new install routine (and ultimately for every old one, too).
I leave it to you though when to tackle that.